File: pset6/mario/more/mario.py
from cs50 import get_int
# Prompt uset for non negative integer inclusive between 1 and 8
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    for j in range(n - (i + 1), 0, -1):
        print(" ", end="")
    for k in range(-1, i):
        print("#", end="")
# Just add two spaces and the same loop as above
    print("  ", end="")
    for k in range(-1, i):
        print("#", end="")
    print()

File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    //Prompt user for non negative integer no greater than 8
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //Draw the half pyramid
    for (int i = 0; i < n; i++)
    {
        for (int s = n - (i + 1); s > 0; s--)
        {
            printf(" ");
        }

        for (int j = -1; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}